### ISSUE TITLE
Cut Firestone replacing all the dirt blocks 

### ISSUE THREAD

**Issue by Grumlin92** [Railcraft/Railcraft#279]
Hi
Earlier today we figured out that it was time to update our server to forge-1.7.10-10.13.0.1199 from on of the old 1.6.4 versions. When we entered the world we where met with massive frame drops (between 1 and 6), and chunks did not load properly. Behind the unloaded chunks we could see some sort of red diamonds (the Cut Firestone). 

We quickly took down the server and removed all the mods, when logging into the world without any mods we found that all the dirt blocks had been removed. 

After swapping the broken world with a backup we started testing out which mod that might be the problem. Every mod was tested by itself on a fresh backup of our world and we quickly narrowed it down to railcraft and that the block causing the trouble was the Cut Firestone.

TL;DR... Cut Firestone replaced all dirt after update.

Edit: After looking through the server log we found this message: 

[23:34:16] [Server thread/INFO] [FML]: Injected new block/item Railcraft:tile.railcraft.firestone.recharge: 202 (init) -> 3 (map).
[23:34:16]
[23:34:16]
